docs/docs/components/reanimated_swipeable.md

```diff
@@ -81,27 +81,29 @@ an argument.
 
 ### `renderLeftActions`
 
-method that is expected to return an action panel that is going to be revealed from the left side when user swipes right.
-This map describes the values to use as inputRange for extra interpolation:
+accepts a function which receives the following arguments:
 
-SharedValue: [startValue, endValue]
+- `progress` value representing swiping progress relative to the width of the returned element.  
+  Equals `0` when `swipeable` is closed, `1` when left action is open.  
+  Linearly increases to `Infinity` as left action overshoots it's open position.
```

Comment:
```suggestion
When an element overshoots the opened position the value tends towards `Infinity`. Goes back to `1` when `swipeable` is released. 
```

Reply:
I think the second sentence is a good addition: ```Goes back to `1` when `swipeable` is released.```.
But for the first one, i think the original works a bit better, it's shorter and i think the wording it uses is a bit clearer

Perhaps the following would work better?
>  As left action `progress` overshoots it's open position, the value linearly increases to `Infinity`.
>  Reverts to `1` when `swipeable` is released.

Reply:
We wanted to make it as close to natural sentence as possible. Also we'd like to avoid `left action`, that's why we used `element`

Reply:
>We wanted to make it as close to natural sentence as possible. Also we'd like to avoid left action, that's why we used element

I believe this is resolved [here](https://github.com/software-mansion/react-native-gesture-handler/pull/3121#discussion_r1782999151)

Reply:
>Am I missing something, or the only thing that has changed is left action?

I'm not sure which PR you're referencing, in [df0ff8c](https://github.com/software-mansion/react-native-gesture-handler/pull/3121/commits/df0ff8cb7be3b0c651b048dced53ffa8b4731b42) both left and right got replaced.

Are you talking about some PR between [df0ff8c](https://github.com/software-mansion/react-native-gesture-handler/pull/3121/commits/df0ff8cb7be3b0c651b048dced53ffa8b4731b42) and [9845471](https://github.com/software-mansion/react-native-gesture-handler/pull/3121/commits/9845471eab4539a1a6d9f4164ab3483a159b296e)?

![image](https://github.com/user-attachments/assets/e2162e08-e4c9-4fa0-90a6-21a6e5bf248a)

Reply:
fixed in [620e247](https://github.com/software-mansion/react-native-gesture-handler/pull/3121/commits/620e247aa4a312d1b649bafe665060594341214f)